pkg/eve/runme.sh

```diff
@@ -59,6 +60,8 @@ Passing -v <local folder>:/out makes sure the file created is given most appropr
 
 -f fmt selects a packaging format: raw (default), qcow2, parallels, vdi and gcp are all valid options.
 
+-o option provides an option to installer cmdline (e.g. -o eve_persist_disk=sda,sdb -o eve_install_disk=sda). May be used for installers.
```

Comment:
Would this example work with sda in two places? Or should it say sdb.sdc for the persist disk?

Reply:
Yes, it will work. In this case we will have /dev/sda9 in the pool along with other disks.